src/components/Loader/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import React from 'react';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person;
+  activeSlug: string | undefined;
+}
+
+export const PersonLink: React.FC<Props> = ({
+  person: {
+    slug,
+    sex,
+    name,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  },
+  activeSlug,
+}) => {
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': activeSlug === slug })}
+    >
+      <td>
+        <Link
+          to={`/people/${slug}`}
+          className={classNames({ 'has-text-danger': sex === 'f' })}
```

Comment:
The className here is conditionally added based on the `sex` property. However, the class name 'has-text-danger' doesn't really describe the condition it's representing. Consider using a more descriptive class name, for example 'is-female'. This makes the code more readable and maintainable.